hutch_python/tests/conftest.py

```diff
@@ -30,3 +31,6 @@ def restore_logging():
     prev_handlers = copy(logging.root.handlers)
     yield
     logging.root.handlers = prev_handlers
+
+
+Experiment = namedtuple('Experiment', ('run', 'proposal'))
```

Comment:
Where is this `namedtuple` used?

Reply:
My fake object that gets created by `load_devices`. I wanted to a full end to end test, so the fake Questionnaire info looks likes
```python
device_class = 'hutch_python.tests.conftest'
args = ['{{run}}', '{{proposal}}']
```
This way when we run `load_devices`, `Experiment` gets imported and instantiated.